appveyor.yml

```diff
@@ -0,0 +1,19 @@
+install:
+  - cinst -y jdk7
```

Comment:
I can try doing this, but this takes ~ 2-3m longer per CI run. appveyor machines have jdk7 pre-installed (this line is technically a noop), but jdk8 takes a while to install.